src/main/java/com/thebois/models/beings/IBeing.java

```diff
@@ -1,19 +1,35 @@
 package com.thebois.models.beings;
 
+import com.thebois.models.IDeepClonable;
 import com.thebois.models.Position;
+import com.thebois.models.beings.roles.AbstractRole;
 
 /**
  * An entity.
  */
-public interface IBeing {
+public interface IBeing extends IDeepClonable<IBeing> {
```

Comment:
Should IBeing really extend IDeepClonable? If AbstractBeing shouldn't be of that type, why should IBeing be? Am I missing something?

Reply:
Nope, you're absolutely right. Have removed it already, will be pushing it up soon.

Reply:
Or actually, it's already pushed. Did you make this comment a few hours ago maybe? 😛

Reply:
what. ok I'm confused.